src/main/java/io/fabric8/jenkins/openshiftsync/BuildSyncRunListener.java

```diff
@@ -258,7 +259,7 @@ private void upsertBuild(Run run, RunExt wfRunExt) {
     }
 
     OpenShiftClient openShiftClient = getOpenShiftClient();
-    String rootUrl = OpenShiftUtils.getJenkinsURL(openShiftClient, namespace);
+    String rootUrl = getHostName(openShiftClient, namespace);
```

Comment:
As I mentioned in PR 12, for a bugzilla we just got from a multi namespace customer, I will be making a change to use the jenkins pod namespace before the build namespace for this ... I'll tag you all in that PR. 

I believe that should be fine with your env var override based on how this is constructed

Reply:
@gabemontero :+1:

Reply:
cool ... for cross referencing, here is the upstream pr:  https://github.com/openshift/jenkins-sync-plugin/pull/198